knowledge-base/adr/0020-generator-options.md

```diff
@@ -51,8 +51,11 @@ We should discuss the details of those defaults and make a concious decision on
 | `--forceOverwrite`                                       | `--overwrite`                    | Overwrite files even it they exist, default false.                                                | _same_                                                                                                                                                       |  false   |
 | `--generateTypedocJson`                                  | `--typedocJson`                  | Writes a default typedoc.json, default true.                                                      | Remove/deprecate, use `files` instead.                                                                                                                       |    -     |
 | **Currently hidden options**                             |
-| `--additionalFiles`                                      | `--files`                        | Copy additional files, identified by glob. Hidden.                                                | _same_ Expose it.                                                                                                                                            |  false   |
+| `--additionalFiles`                                      | `--include`                      | Copy additional files, identified by glob. Hidden.                                                | _same_ Expose it.                                                                                                                                            |  false   |
```

Comment:
We have discussed `files` for this. I would like to propose `include`. Let me know what you think.

Reply:
@FrankEssenberger @jjtang1985

Reply:
For me `--include` is fine since it is followed by a path than so`--include select/something/via/Glob` makes sense.